### DISCUSSION TITLE
Feature request: show read receipts after sending a message

### DISCUSSION THREAD

**Discussion by Oblivious02** [rodit/SnapMod#179] [Ideas & Feature Suggestions]
I usually use "Hide read receipts" but the problem is even after sending a message you still have a notification to view the message.

Either than that the module is really flawless, thanks!

**Comment by Magna-Graecia** [comment]
I see what you're saying, but it's better than SnapFreedom had where even trying to type a message with Stealth enabled would cause Snapchat to crash. It was good in the sense that it prevented you from giving away your stealth in case you forgot that you had it on and sent a message anyway, but it was bad in the sense that it gave you less flexibility like you have with SnapMod.

I would suggest enabling Stealth Mode on a per-chat basis and leaving all of the global stealth features disabled. The only ones I keep enabled are Hide Screenshot/Recording Notifications, Hide Saved to Camera Roll, and the 2 related to anonymous story viewing. The rest I enable per-chat as needed.

What you can also do if you want to read messages in stealth, but then reply to them with having a read receipt shown is just use the message preview feature. This is what I do when I want to quickly read a message without the recipient knowing, but also have the ability to immediately go into the chat and cause a read receipt to be sent and send a reply. This way I don't need to keep toggling Stealth Mode on and off. This is what all of these features are for. You could also enable Show Message in Notifications, but that one may be less convenient since it will only show you the latest message received in a notification, and not the last 100 messages both sent/received like you can set the message preview to do.
